src/forms/StudyFormSteps.js

```diff
@@ -259,56 +318,121 @@ export const GrantStep = ({
         handleChange={handleChange}
         handleBlur={handleBlur}
         handleFocus={id => setFocused(id)}
+        readOnly={!editing && !newStudy}
       />
-      <FormField
-        id="description"
-        name="Description"
-        description="Study description in markdown, commonly the X01 abstract
+      {!editing && !newStudy ? (
+        <>
+          <Header as="h5">Description:</Header>
+          <Segment
+            attached={values.description.length > 0}
+            className={foldDescription ? 'max-h-500' : null}
+          >
+            {values.description && values.description.length > 0 ? (
+              <Markdown
+                source={values.description}
+                renderers={{
+                  image: Image,
+                  table: props => <Table>{props.children}</Table>,
+                  list: List,
+                  listItem: List.Item,
+                }}
+              />
+            ) : (
+              <Header disabled className="mt-6" as="h5">
+                No study description available.
+              </Header>
+            )}
+          </Segment>
+          {values.description && values.description.length > 0 && (
+            <Button
+              attached="bottom"
+              onClick={() => setFoldDescription(!foldDescription)}
+              className="mb-15"
+            >
+              {foldDescription ? 'Read More' : 'Read Less'}
```

Comment:
🤔 this should also check the character length so that it only shows when there is overflowing text, no need to show it with text that fits the readOnly size of the box

Reply:
Because we are displaying images and tables here too, those may take a lot of height but with a small number of characters. So we trying to have the read more / read less button there for all cases.